channels/rail/client/rail_main.c

```diff
@@ -97,9 +97,26 @@ UINT rail_send_channel_data(railPlugin* rail, void* data, size_t length)
  */
 static void rail_client_clean_exec_order(RAIL_EXEC_ORDER* exec)
 {
-	free(exec->exeOrFile.string);
-	free(exec->workingDir.string);
-	free(exec->arguments.string);
+    if (!exec)
+        return;
+
+    if (exec->exeOrFile.string)
```

Comment:
as suggested, I added these NULL checks. please give some tips if there are better ways to do so.